src/box/vy_scheduler.c

```diff
@@ -1996,47 +2019,45 @@ vy_task_complete(struct vy_task *task)
 	return -1;
 }
 
+static void
+vy_scheduler_finish_tasks(struct vy_scheduler *scheduler, int *tasks_done,
```

Comment:
Nit: `vy_scheduler_complete_tasks` might be a better name.

Reply:
Done.

```diff
diff --git a/src/box/vy_scheduler.c b/src/box/vy_scheduler.c
index 8818bb5e7..471db2c60 100644
--- a/src/box/vy_scheduler.c
+++ b/src/box/vy_scheduler.c
@@ -2020,7 +2020,7 @@ vy_task_complete(struct vy_task *task)
 }

 static void
-vy_scheduler_finish_tasks(struct vy_scheduler *scheduler, int *tasks_done,
+vy_scheduler_complete_tasks(struct vy_scheduler *scheduler, int *tasks_done,
                          int *tasks_failed)
 {
        *tasks_done = 0;
```